hack/runner/webhook/aws.go

```diff
@@ -110,7 +109,7 @@ func deleteEc2InstanceByName(client *ec2.EC2, uniqueID string) error {
 			{
 				Name: aws.String("tag:Name"),
 				Values: []*string{
-					aws.String(strings.Join([]string{"*", uniqueID, "*"}, "")),
```

Comment:
Just curious, was this not valid from the linter? Seems perfectly reasonable to me (although it is using a literal string slice)

Reply:
Huh! Interesting! Thanks for the thoughts!!